File: api/code-samples/javascript/klaytnDebug/blockchainInspection/startContractWarmUp.js
const OpenSdk = require("opensdk-javascript");

(() => {
    const sdk = new OpenSdk(new OpenSdk.ApiClient("https://api.baobab.klaytn.net:8651"));

    const address = "0x4Cd39B49064a31E966cA0ddF4111aCe2eD7E9502"

    sdk.debug.startContractWarmUp(address, {}, (err, data, response) => {
        console.log(data);
    });
}
)()

File: api/code-samples/python/klaytnDebug/blockchainInspection/startContractWarmUp.py
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

address = "0x4Cd39B49064a31E966cA0ddF4111aCe2eD7E9502"

sdk = OpenSDK(host)
debug_response = sdk.debug.start_contract_warm_up(address)

print(debug_response)
